Description: nscd profile: allow paranoia mode
 In /etc/nscd.conf there is an option allowing to restart nscd after a
 certain time. However, this requires reading /proc/self/cmdline -
 otherwise nscd will disable paranoia mode.
 .
 References: https://bugzilla.opensuse.org/show_bug.cgi?id=971790
 .
 Acked-By: Jamie Strandboge <jamie@canonical.com> for trunk, 2.10 and 2.9
Origin: commit, revision id: apparmor@cboltz.de-20160321203019-wbbknrxajgqkqhjq
Author: Christian Boltz <apparmor@cboltz.de>
Last-Update: 2016-03-21
X-Bzr-Revision-Id: apparmor@cboltz.de-20160321203019-wbbknrxajgqkqhjq

=== modified file 'profiles/apparmor.d/usr.sbin.nscd'
--- old/profiles/apparmor.d/usr.sbin.nscd	2014-12-01 22:44:13 +0000
+++ new/profiles/apparmor.d/usr.sbin.nscd	2016-03-21 20:30:19 +0000
@@ -31,6 +31,7 @@
   /{var/cache,var/run,run}/nscd/{passwd,group,services,hosts,netgroup} rw,
   /{,var/}run/{nscd/,}nscd.pid rwl,
   /var/log/nscd.log rw,
+  @{PROC}/@{pid}/cmdline r,
   @{PROC}/@{pid}/fd/ r,
   @{PROC}/@{pid}/fd/* r,
   @{PROC}/@{pid}/mounts r,

